Source/cmake/OptionsWPE.cmake

```diff
@@ -157,9 +157,10 @@ if (USE_SOUP2)
     set(SOUP_API_VERSION 2.4)
     set(WPE_API_VERSION 1.0)
 else ()
+    SET_AND_EXPOSE_TO_BUILD(ENABLE_2022_GLIB_API ON)
     set(SOUP_MINIMUM_VERSION 3.0.0)
     set(SOUP_API_VERSION 3.0)
-    set(WPE_API_VERSION 1.1)
+    set(WPE_API_VERSION 2.0)
```

Comment:
Is the assumption that nobody is using wpewebkit with soup3 accurate? I think it would be safer to simply add a new api version like for GTK.

Reply:
That's fine, but then I need to expose ENABLE_2022_GLIB_API as a public build option for WPE. @aperezdc, is this OK?

After previous discussions with @aperezdc, I gathered that he was primarily concerned about maintaining API stability for libsoup 2 API for a short while longer. My understanding is that WPE is OK with API breaks much more frequently than GTK is, and therefore we'll soon be able to remove everything except the new WPE API version.

Reply:
BTW this was previously decided in https://bugs.webkit.org/show_bug.cgi?id=242407, but it seems harmless to change our minds and keep it for a little while.

Reply:
Need a final decision on this, please.

Reply:
For now we'll keep both 1.0 and 1.1 APIs. I think they should both be removed soon as it should be easy to port to 2.0, but Igalia can do that when desired.

I will make 2.0 the default API and add a new option ENABLE_WPE_1_1_API to get the old version.